### ISSUE TITLE
List: Render using <ol>, <ul>, <li>

### ISSUE THREAD

**Issue by hross** [OfficeDev/office-ui-fabric-react#1253]
### Bug Report

Because of the way lists are currently implemented (paginated divs), I cannot render semantic ordered or unordered lists with the component.

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes, maybe

Requested priority: High-ish (?)

#### Describe the issue:

This would arguably be a lower priority feature (although I think we should definitely provide a way to render virtualized lists with the List component) except that screen readers in document mode typically use semantic lists of elements to help them navigate the page. In order to provide a decent SR experience, at least having the option to render an ordered list should be part of this component.

#### Actual behavior:

divs are rendered.

#### Expected behavior:

We should render an ol/ul with an li underneath. The tricky part is that ol/ul cannot have any children other than li elements which means the current div paging model would need to be rewritten (and possibly take a performance hit) so that it uses first and last elements in a page instead of a container element.

**Comment by hross** [comment]
I wonder if we can do this: http://stackoverflow.com/questions/11131234/the-reason-to-use-role-list-and-role-listitem

This appears to work in NVDA: https://patrickhlauke.github.io/aria/demos/role-list.html

**Comment by hross** [comment]
This example works in NVDA, using list:

http://codepen.io/pogostick_snowman/pen/VpzaoK

**Comment by cschleiden** [comment]
@hross Do you think we still need this or are your changes good enough?

**Comment by hross** [comment]
I think we are going to have to revisit this, unfortunately (see ongoing TFS discussion email threads :)). I will file something additional when we know what we are looking for.
